vcs/util/path.go

```diff
@@ -0,0 +1,12 @@
+package util
+
+import "strings"
+
+// Rel strips the leading "/" prefix from the path string, effectively turning
+// an absolute path into one relative to the root directory.
```

Comment:
Fair points. I've documented the special case and moved the `Rel` function into a new `vcs/internal` package.

I think `vcs/internal` is more canonical than `vcs/internal/util` (we probably won't have more than one internal package, given that it only contains one function at this point).